features/old/accounts/password_change.feature

```diff
@@ -8,13 +8,12 @@ Feature: Password change
 
     Given current domain is the admin domain of provider "foo.example.com"
     When I log in as provider "foo.example.com" with password "supersecret"
-    When I follow "Account Settings"
-    And I follow "Personal Details"
+    When I go to the provider personal details page
     And I fill in "Password" with "monkey"
     And I fill in "Current password" with "supersecret"
     And I press "Update Details"
 
-    When I follow "Sign Out"
+    When I go to provider logout
```

Comment:
Small change but better `I logout`?

Reply:
That won't work because it needs JS now in order to be rendered

Reply:
the step is 
```
When /^I log ?out$/ do
  visit '#session-menu'
  click_link 'Sign Out' || 'Log Out'
end
```
And `#session-menu` doesn't exist

Reply:
Then change the step ;)